service/src/main/resources/pipelines-config.yml

```diff
@@ -1,3 +1,12 @@
+# This env section is for deployment-specific values
+env:
+  pipelines:
+    imputation:
+      1:
+        # the default currently points to the GCP dev workspace teaspoons-imputation-dev/teaspoons_imputation_dev_storage_workspace_20240726
+        storageWorkspaceContainerUrl: ${IMPUTATION_STORAGE_WORKSPACE_STORAGE_URL:gs://fc-secure-10efd4d7-392a-4e9e-89ea-d6629fbb06cc}
```

Comment:
so this isnt actually versioning the values coming from helmfile.  the value is still beign passed with `IMPUTATION_STORAGE_WORKSPACE_STORAGE_URL` so if we wanted to have a different value for v2 of array imputation what would the name of that variable be?